config/lucid.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+return [
+    /*
+     |--------------------------------------------------------------------------
+     | Dashboard
+     |--------------------------------------------------------------------------
+     |
+     | By default /lucid/dashboard is available on when env('APP_DEBUG')
+     | is true. If you set this value to "true" it will be always accessible
+     | even on production environment.
+     |
+     */
+    'dashboard'     => null,
+
+    /*
+     |--------------------------------------------------------------------------
+     | Microservice
+     |--------------------------------------------------------------------------
+     |
+     | By default Lucid Architecture is setup as microservice. Microservice
+     | means that you will have only one service and all features / jobs / data
+     | etc. will be generated into /app/ directory.
+     |
+     | When you set this value to "false" it means you application have multiple
+     | services and all classes will be generated into /src/ directrory.
+     | After changing that value all "lucid" classes should be moved
+     | from /app/ to /src/.
+     |
+     */
+     //'microservice'  => true,
```

Comment:
Actually this have a lot of possibilities. As I see it, with lucid.php you should be able to set /src/ path or /app/ namespaces etc. This config file can be shared with `foundation` package.   Instead of some `Finder` logic you can simply check config var. 

For now we can remove this part of course and you can consider is as idea to think about.